client/buildpacks.go

```diff
@@ -161,6 +155,23 @@ func (b *buildpacksFunctionServer) run() (func(), error) {
 	}, nil
 }
 
+func (b *buildpacksFunctionServer) getRuntimeVars() []string {
+	var variables = strings.Split(b.envs, ",")
```

Comment:
[nit] prefer `:=` for initializing with values and move closer to the loop where it's used